test/test.ml

```diff
@@ -44,7 +44,7 @@ let test_primitives () =
   Alcotest.(check (result bool error))
     "(of_yaml) same bool" (Ok correct_bool_of) test_bool_of
 
-type person = { name : string; age : int } [@@deriving yaml]
+type person = { name : string; [@default "Alice"] age : int } [@@deriving yaml]
```

Comment:
Is the semicolon between fields misplaced here? 

`{ name : string [@default "Alice"]; age : int }`

Reply:
Hi @ayc9 Thank you for the review. 

I don't think the semicolon is misplaced. The semicolon ended the ```name field``` and the ```default``` attribute is attached to that field.

It's looks more clearly if the code is broken down like this 
```
type person = { 
  name : string; [@default "Alice"] 
  age : int;
} [@@deriving yaml]